src/App.tsx

```diff
@@ -1,5 +1,79 @@
+import styles from "./App.module.css";
+
 function App() {
-  return <div>여기다 작업하세요!</div>;
+  return (
+    <div className={styles.container}>
+      <div className={styles.wrap}>
+        <img src="/class.png" alt="" />
+        <div className={styles.text1}>
+          UI/UX 단기간 속성 강의 (Figma, 6주차 단기 과정)
+        </div>
+        <div className={styles.text2}>
+          카카오, 배달의 민족 등 10년차 현직 디자이너 강사와 함께하는 6주차
+          과정의 UI/UX 단기간 속성 강의
+        </div>
+      </div>
+      <div className={styles.notices}>
+        <button className={styles.noti}>
+          <img src="/alarm.png" alt="" />
+          <div className={styles.text_wrap}>
```

Comment:
```suggestion
          <div className={styles.textWrap}>
```

이런식으로 합성어를 클래스명으로 할 텐데는 kebab-case(`text-wrap`)와 camelCase(`textWrap`)의 두 가지 방법을 많이 사용합니다. 저희처럼 css modules를 사용할 때는 후자를 많이 쓰고, 저도 그런 편입니다.